client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -0,0 +1,361 @@
+        name: 'fourth',
+    },
+    {
+        id: 4,
+        name: 'אחר',
+    },
+];
+
+const backgroundIllnessesList: Check[] = [
+    {
+        id: 0,
+        name: 'one',
+    },
+    {
+        id: 1,
+        name: 'two',
+    },
+    {
+        id: 2,
+        name: 'three',
+    },
+    {
+        id: 3,
+        name: 'four',
+    },
+    {
+        id: 4,
+        name: 'אחר',
+    },
+
+];
+
+const hospitals: string[] = ['שיבא', 'איכילוב', 'אסף הרופא'];
+
+const ClinicalDetails: React.FC = (): JSX.Element => {
+    const classes = useStyles();
+
+    const [isInIsolation, setIsInIsolation] = React.useState<boolean>(false);
+    const [hasSymptoms, setHasSymptoms] = React.useState<boolean>(false);
+    const [isUnkonwnDateChecked, setIsUnkonwnDateChecked] = React.useState<boolean>(false);
+    const [hasBackgroundIllnesses, setHasBackgroundIllnesses] = React.useState<boolean>(false);
+    const [wasHospitalized, setWasHospitalized] = React.useState<boolean>(false);
+    const [otherSymptom, setOtherSymptom] = React.useState<string>('');
+    const [otherBackgroundIllness, setOtherBackgroundIllness] = React.useState<string>('');
+    const [selectedSymptoms, setSelectedSymptoms] = React.useState<string[]>([]);
+    const [selectedBackgroundIllnesses, setSelectedBackgroundIllnesses] = React.useState<string[]>([]);
+    const { isInIsolationToggle, hasSymptomsToggle, hasBackgroundIllnessesToggle, wasHospitalizedToggle } = useClinicalDetails(
+        {
+            setIsInIsolation, setHasSymptoms, setHasBackgroundIllnesses, setWasHospitalized
+        });
+
+    const handleUnkonwnDateCheck = () => {
+        setIsUnkonwnDateChecked(!isUnkonwnDateChecked);
+    };
+
+    const updateClinicalDetails = (fieldToUpdate: ClinicalDetailsFields, updatedValue: any) => {
+        context.setClinicalDetailsData({...context.clinicalDetailsData as ClinicalDetailsData, [fieldToUpdate]: updatedValue});
+    };
+
+    const handleSymptomCheck = (symptom: Check) => {
+        if (selectedSymptoms.find(checkedName => checkedName === symptom.name)) {
+            setSelectedSymptoms(selectedSymptoms.filter((checkedSymptom) => checkedSymptom !== symptom.name));
+        } else {
+            selectedSymptoms.push(symptom.name);
+        };
+
+        updateClinicalDetails(ClinicalDetailsFields.SYMPTOMS, selectedSymptoms)
+    };
+
+    const handleBackgroundIllnessCheck = (backgroundIllness: Check) => {
+        if (selectedBackgroundIllnesses.find(checkedName => checkedName === backgroundIllness.name)) {
+            setSelectedBackgroundIllnesses(selectedBackgroundIllnesses.filter((checkedBackgroundIllness) => checkedBackgroundIllness !== backgroundIllness.name));
+        } else {
+            selectedBackgroundIllnesses.push(backgroundIllness.name);
+        };
+
+        updateClinicalDetails(ClinicalDetailsFields.BACKGROUND_ILLNESSES, selectedBackgroundIllnesses)
+    };
+
+    const context = React.useContext(clinicalDetailsDataContext);
```

Comment:
Move up, isn’t it creating new context?

Reply:
it is

Reply:
so how is it updating the context you provided

Reply:
its the same context used in investigationForm